src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -594,7 +594,13 @@ export class ExtensionEditor extends BaseEditor {
 	private renderSettings(container: HTMLElement, manifest: IExtensionManifest, onDetailsToggle: Function): boolean {
 		const contributes = manifest.contributes;
 		const configuration = contributes && contributes.configuration;
-		const properties = configuration && configuration.properties;
+		let properties = configuration && configuration.properties;
```

Comment:
May be we should fix the type of `contribues.configuration` to be `IConfiguration | IConfiguration[]` and the consumers can check for the type?

Reply:
Updated the type.